sys/mips/cheri/cheri.c

```diff
@@ -73,7 +73,6 @@ CTASSERT(sizeof(struct cheri_frame) == (34 * CHERICAP_SIZE));
  * that work on other architectures might break alignment on CHERI.
  */
 CTASSERT(offsetof(struct trapframe, ddc) % CHERICAP_SIZE == 0);
-CTASSERT(offsetof(struct thread, td_cheri_mmap_cap) % CHERICAP_SIZE == 0);
```

Comment:
It did not seem worth it to find a different field in struct thread to replace this assertion with.  At this point, many things would fall over if the toolchain got this wrong, and we can probably trim many of the assertions from this file (e.g. `sizeof(void *)`, etc.)

Reply:
I think these date to not really having a working compiler.